datajoint/table.py

```diff
@@ -73,6 +73,182 @@ def declare(self, context=None):
         else:
             self._log('Declared ' + self.full_table_name)
 
+        def preview_alter(self,new_definition,_preview=True):
+        """
+        Returns changes required to go from current defition to new defition.
+        Does not execute any changes. Use alter() method to finalize any changes.
+        Only supports table comment, and SECONDARY attribute changes(ADD,DROP,CHANGE).
+        Reordering currently not supported.
+        Use # {<old_name>} <comment> to rename given attribute.(insert values between '< >') 
+
+        :param new_definition: string, same format used to create the table.
+        :param _preview: True - Human friendly preview of changes. False - MYSQL ALTER preview.
+        :return: preview changes based on _preview. 
+        """
+
+        output_str = '' #user-friendly preview
```

Comment:
Let's drop the user-friendly preview for now.  The SQL code should be sufficient to present to user for review.

Reply:
removed